packages/calcite-components/src/components/tooltip/tooltip.e2e.ts

```diff
@@ -388,7 +388,37 @@ describe("calcite-tooltip", () => {
 
     await page.evaluate(() => {
       const ref = document.getElementById("ref");
-      ref.click();
+      const event1 = new Event("click", {
```

Comment:
Just curious, but is there a specific reason why `MouseEvent` or `PointerEvent` isn't being used here?

Reply:
Nope, will update.